sat/cli/jobstat/main.py

```diff
@@ -0,0 +1,64 @@
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+"""
+The main entry point for the jobstat subcommand.
+"""
+import logging
+from sat.report import Report
+from sat.config import get_config_value
+from sat.apiclient import APIError
+from sat.apiclient.JobstatStateChecker import JobstatStateChecker
+from sat.session import SATSession
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def do_jobstat(args):
+    """Run the jobstat command with the given arguments.
+    Args:
+        args: The argparse.Namespace object containing the parsed arguments
+            passed to this subcommand.
+    """
+    state_checker_client = JobstatStateChecker(SATSession())
+    try:
+        application_data = state_checker_client.get_all()
+        if not application_data:
+            return
+        report = Report(
+            tuple(application_data[0].keys()),
+            None,
+            args.sort_by,
+            args.reverse,
+            get_config_value("format.no_headings"),
+            get_config_value("format.no_borders"),
```

Comment:
FWIW, these arguments aren't needed. When the `no_headings` and `no_borders` arguments are `None` (the default), the `Report` class looks up the config file values. See here: https://github.com/vinaybabug-hpe/sat/blob/993cf683b89798958244ac2ab4473f5e8650bef0/sat/report.py#L102-L114

Reply:
removed them